### DISCUSSION TITLE
How to update the class for the actions column

### DISCUSSION THREAD

**Discussion by Brav** [Power-Components/livewire-powergrid#896] [Q&A] [answered]
I am using latest Livewere Powergrid (v 3), and I can't find anywhere how to change/update actions column classes. Is it even possible to do this?

**Comment by abhaypithadiya** [comment, marked as answer]
Hey!

Publish the Powergrid view file using:
```php artisan vendor:publish --tag=livewire-powergrid-views```

Go to resources\views\vendor\livewire-powergrid\components\actions.blade.php

Here you can change and update column classes.

Hope this helps!
